pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -79,9 +79,10 @@ func (p *v1PodResourcesServer) List(ctx context.Context, req *v1.ListPodResource
 		podResources[i] = &pRes
 	}
 
-	return &v1.ListPodResourcesResponse{
+	response := &v1.ListPodResourcesResponse{
```

Comment:
I don't have strong feelings here but IIRC it was mentioned indeed as non-blocking cheange

Reply:
yes it was requested here [1] 
[1] https://github.com/kubernetes/kubernetes/pull/115847#discussion_r1133989554

Reply:
ok, I see you changed all the instance in this file. SGTM.